src/Microsoft.OData.Client/DataServiceClientRequestMessageArgs.cs

```diff
@@ -46,6 +47,35 @@ public DataServiceClientRequestMessageArgs(string method, Uri requestUri, bool u
             }
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DataServiceClientRequestMessageArgs"/> class.
+        /// </summary>
+        /// <param name="method">Method of the request.</param>
+        /// <param name="requestUri">The Request Uri.</param>
+        /// <param name="useDefaultCredentials">True if the default credentials need to be sent with the request. Otherwise false.</param>
+        /// <param name="usePostTunneling">True if the request message must use POST verb for the request and pass the actual verb in X-HTTP-Method header, otherwise false.</param>
+        /// <param name="headers">The set of headers for the request.</param>
+        /// <param name="client">The HttpClient Instance for the request</param>
+        public DataServiceClientRequestMessageArgs(string method, Uri requestUri, bool useDefaultCredentials, bool usePostTunneling, IDictionary<string, string> headers, HttpClient client)

```

Comment:
Can this constructor call the other existing constructor (or vice versa) instead of duplicating the initialization statements